test/cypress/integration/payment-links/create-payment-link.cy.test.js

```diff
@@ -204,6 +204,33 @@ describe('The create payment link flow', () => {
         cy.get('button').should('exist').should('contain', 'Create payment link')
       })
 
+      it('should show reporting column component', () => {
+        cy.get('a#add-reporting-column').should('exist').should('contain', 'Add a reporting column')
+        cy.get('a#add-reporting-column').click()
+        cy.location().should((location) => expect(location.pathname).to.eq('/create-payment-link/add-reporting-column'))
+      })
+
+      it('should reject empty values and accept valid input', () => {
```

Comment:
I was thinking we could split this up into 2 tests?

Reply:
Happy to split this up